env-setup.bat

```diff
@@ -0,0 +1,9 @@
+ldc2-1.35.0-windows-x64\bin\dub.exe fetch dlib
+ldc2-1.35.0-windows-x64\bin\dub.exe fetch dlib:math
+ldc2-1.35.0-windows-x64\bin\dub.exe fetch dlib:math:portable
+
+ldc2-1.35.0-windows-x64\bin\dub.exe build
+
+REM ldc2-1.35.0-windows-x64\bin\dub.exe add dlib
+REM ldc2-1.35.0-windows-x64\bin\dub.exe add dlib:math
+REM ldc2-1.35.0-windows-x64\bin\dub.exe add dlib:math:portable
```

Comment:
the whole point of this file is now entirely debatable.

since five repos are necessary to get a working build:
```
sacengine
dagon
dlib
DerelictGL3
DerelictSDL2
```

and since dlib is one of the repositories needed - apart from the fact that this file tried fixing what recursive cloning should have done - it's useless to have it sitting around causing confusion (and only breaking the project as it will definitelly suggest to update/upgrade dub, and subsequently screw with the dependency versions)

Reply:
deleted. resolving.